integrations/tensorflow/test/iree_tfl_tests/update_tflite_model_documentation.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Updates the TFLite integration test model documentation.
+
+When changes are made to any models in this directory, please run this script to
+update the dashboard of models.
+"""
+
+import os
+from pathlib import Path
+from typing import Sequence
+
+# The symbols to show in the table if the operation is supported or not.
+SUCCESS_ELEMENT = '<span class="success-table-element">✓</span>'
+FAILURE_ELEMENT = '<span class="failure-table-element">✗</span>'
```

Comment:
```suggestion
# The symbols to show in the table if the operation is supported or not.
SUCCESS_ELEMENT = 'PASS ✓'
FAILURE_ELEMENT = 'FAIL ✗'
```

clarifying my comment. The `<span></span>` inline HTML is only necessary to apply a CSS class, but GitHub's markdown renderer does not support CSS. We used that before since the pages were published to a website, where we could set text color and other styling through the `success-table-element` and `failure-table-element` css classes.

Reply:
I see, thank you!